containers/daap-presigned-url/tests/unit/mock_test.py

```diff
@@ -58,11 +64,15 @@ def test_dataproduct_does_not_exist(s3_client, fake_context, region_name, monkey
     )
 
     event = {
-        "queryStringParameters": {
-            "database": database,
-            "table": table,
-            "contentMD5": "3f92d72f7e805b66db1ea0955e113198",
-        }
+        "pathParameters": {
+            "data-product-name": database,
+            "table-name": table,
+        },
+        "body": json.dumps(
```

Comment:
Looking at the tests it seems like the API gateway must be passing the deserialized version of the body, and you can drop the `json.dumps` here.

Other than that all looks good to me 👍🏻

Reply:
Thanks Mat. Yes I just ran into this in the tests and removed it. Many Thanks